### PR TITLE
Added support to render KaTex in Markdown

### PR DESCRIPTION
**Pull request by eswarasai**
### Description

- [x] Added support to render KaTex in Markdown as per docs suggested here -- https://mdxjs.com/guides/math-blocks

### Steps
- Change Markdown extension to `MDX`
- Add path to KaTex css file on the top of MDX file
- Write KaTex formulas that's needed to be rendered within the Markdown file

### Screenshot
katex

Fixes: #128

### CODE REVIEW

File: docs/cent-node/overview/introduction.mdx [hunk 1]
```diff
@@ -7,12 +7,15 @@ redirect_from:
   - /cent-node/
 ---
 
+<!-- Apply katex css -->
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.11.0/dist/katex.min.css" integrity="sha384-BdGj8xC2eZkQaxoQ8nSLefg4AV4/AwB3Fj+8SUSo7pnKP6Eoy18liIKTPn9oBYNG" crossOrigin="anonymous"/>
```
Comment (lucasvo) [inline docs/cent-node/overview/introduction.mdx]: Can we not have this in the mdx but have it be injected automatically outside of the content?

File: src/components/Footer/index.js [hunk 2]
```diff
@@ -14,11 +13,11 @@ export const JoinSlack = () => {
 }
 
 
-const Address = styled.address`
+/*const Address = styled.address`
```
Comment (lucasvo) [inline src/components/Footer/index.js]: Why is this needed?

File: docs/cent-node/overview/introduction.mdx (hunk 1 above)
Comment (lucasvo) [reply docs/cent-node/overview/introduction.mdx]: Also, I'd prefer not to load this from a CDN but package it into our existing JS bundle.

File: src/components/Footer/index.js (hunk 2 above)
Comment (eswarasai) [reply src/components/Footer/index.js]: I've just commented the existing code. Can remove it in case we don't need it.

Comment (lucasvo) [reply src/components/Footer/index.js]: pls delete this code
